addOns/sequence/src/main/java/org/zaproxy/zap/extension/sequence/automation/SequenceActiveScanJob.java

```diff
@@ -255,19 +256,33 @@ private static List<SequenceStepData> scanSequence(
                 new StdActiveScanRunner(
                         script, contextWrapper.getContext(), user, contextSpecificObjects);
 
+        Stats.incCounter(ExtensionSequenceAutomation.STATS_PREFIX + "ascan.scan");
```

Comment:
Is it possible to move (some of) these stats into the StdActiveScanRunner?
Right now we're not getting any stats if the scan is run from the desktop..

Reply:
I'll take a look.